### DISCUSSION TITLE
superset-frontend build

### DISCUSSION THREAD

**Discussion by yashu183** [apache/superset#22472] [Q&A / Help]
I cloned the github repo and running `npm run build` in `superset-frontend` directory
There were so many linting issues and `eslint --fix` or `npm run lint` cant fix them

#### How to reproduce the bug

1. clone the official giuthub repo
2. cd superset-frontend
3.Run `npm i` and `npm run build`

### Expected results

I expect to build the frontend successfully

**Comment by rusackas** [comment]
@yashu183 there's definitely a LOT of noise in there, for sure. As for the linting, they're all basically warnings that we're chipping away at as technical debt, but they should all be warnings, not errors, and the frontend _should_ be building successfully.

Are your builds failing? If so, this is definitely an issue!

If the build is working, is that a particular flavor of warning/lint rule that you'd like to chip away at with this issue? I fear that "clean up all the errors" is not a bug, but more of an ongoing hill to climb. If you'd like to tackle any of this work with us, please feel free to join the #debug-together or #contributing channels on [Slack](https://join.slack.com/t/apache-superset/shared_invite/zt-1jp6hjzrq-H0PlFtToyLWuPiJDuRWCNw) and we can coordinate the effort a little :)

**Comment by yashu183** [comment]
Hii @rusackas 
Thanks for the reply
Yeah, builds are failing even when I run `docker build .` to build my custom image, it's failing on `npm run build`

I am already there on slack channels

**Comment by rusackas** [comment]
OK, docker build is a separate process, so we'll focus on the npm (webpack, frontend) builds first. Can you give this issue a more descriptive/specific title and let us know specifically what error(s) are causing the build to fail? Without those details, this is fairly inactionable. Also, are you based off of current/recent `master` branch, or some other tag/branch/fork?

**Comment by yashu183** [comment]
Hii,
I am running the build in the master branch
<img width="946" alt="Screenshot_20221215_100414" src="https://user-images.githubusercontent.com/70622860/207938266-f5a7d8f8-e2da-41e2-858a-a2fb74e3d1c7.png">
Its getting stopped at this module everytime and then this happens....
<img width="946" alt="Screenshot_20221215_102156" src="https://user-images.githubusercontent.com/70622860/207938280-108b3042-d44a-4d76-ac79-4edab89d2342.png">

**Comment by rusackas** [comment]
It looks like you have hundreds of thousands of linting/prettier errors... did you run your files through something, or are you using an unsupported (i.e. windows) platform?

**Reply by yashu183** [reply]
Yeah I was using a windows machine

**Comment by rusackas** [comment]
This issue doesn't seem to be an actual bug on master, or the whole world would be complaining about it. I think you have a platform/configuration error, or need to just run `npm run lint-fix` from the `superset-frontend` directory. I might be wrong about these diagnoses, but I'm moving this to a Q&A thread since it's more troubleshooting than bug reporting, I think.
